node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Hey, @maudem. We should figure out how these yarn files got created in your workspace when they're not checked in. I wonder if your using a different yarn version?

Reply:
Okie dokie. I'm using version 1.22.10